python/pyspark/sql/tests.py

```diff
@@ -2175,7 +2175,7 @@ def test_replace(self):
 
         # replace with subset specified by a string of a column name w/ actual change
         row = self.spark.createDataFrame(
-            [(u'Alice', 10, 80.1)], schema).replace(10, 20, subset='age').first()
+            [(u'Alice', 10, 80.1)], schema).replace(10, 'age', value=20).first()
```

Comment:
Will this conflict with the convention of function argument in Python?

Usually, I think the arguments before keyword arg are resolved by position. But now `age` is resolved to `subset` which is the third argument behind `value`.

Since the function signature is changed, this may not be a big issue.